### PR TITLE
Adding climate4r new packages

### PR DESCRIPTION
**Pull request by cofinoa**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-climate4r.climdex/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-climate4r.climdex
+  version: {{ version|replace("-", "_") }}
+
+source:
+  git_url: https://github.com/SantanderMetGroup/climate4R.climdex
+  git_tag: v0.2.2
```
Comment (dbast) [inline recipes/r-climate4r.climdex/meta.yaml:11]: tags are mutable and thus not secure... can you change that to e.g.
```suggestion
  url: https://github.com/SantanderMetGroup/climate4R.climdex/archive/refs/tags/v{{ version }}.tar.gz
  sha256: dce447218dc57730870c68af01291179acf9f4ae67a780cc3bea3a05a9dbbe51
```

Same for all the other packages specified like this.

**Comment by cofinoa** [comment]
@dbast I'm facing an issue with some dependencies that were resolved some months ago from the 'r' channel, but now they are not found. 

The problem is with 'r-sticky' and 'r-deepnet', they exist on the 'r' channel, but the this channel it's not been used by chec process.

**Comment by dbast** [comment]
@cofinoa All dependencies have to be available via the conda-forge channel (dependencies in r/defaults don't help)

Additionally some updates to be done:
* remove the packages that already exist on conda-forge
* update recipes to not use git tags.

File: recipes/r-downscaler/conda_build_config.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+channel_sources:
+  - conda-forge,r,defaults
```
Comment (dbast) [inline recipes/r-downscaler/conda_build_config.yaml:2]: Don't do this... packages in r/defaults are build against older R versions, so don't help here.

**Comment by dbast** [comment]
This looks much better now. See the inline comment and please use stable URLs + sha256 for all recipes... (not git_urls + tag).

**Pull request closed without merging**